### PR TITLE
ARCH-283: Use suppress logging in jwt_decode_handler.

### PR DESCRIPTION
**Pull request by robrap**
Temporarily, ecommerce is delegating to the edx-drf-extensions
jwt_decode_handler, with its own fallback to allow for multiple issuers.
Logging suppression inside the edx-drf-extensions jwt_decode_handler is
required until ecommerce is down to a single issuer and can use the
edx-drf-extensions jwt_decode_handler directly.

Note: this should be reverted as part of ARCH-276.

ARCH-283

### CODE REVIEW

File: ecommerce/extensions/api/tests/test_handlers.py [hunk 1]
```diff
@@ -85,8 +85,9 @@ def test_decode_success_edx_drf_extensions(self, mock_set_custom_metric):
     )
     @override_switch('jwt_decode_handler.log_exception.edx-drf-extensions', active=True)
     @mock.patch('edx_django_utils.monitoring.set_custom_metric')
+    @mock.patch('edx_rest_framework_extensions.auth.jwt.decoder.logger')
```
Comment (robrap) [inline ecommerce/extensions/api/tests/test_handlers.py:88]: Reviewers: The original unit test was checking the wrong logger.  I ensured that this test fails with the old edx-drf-extensions and passes when requesting to suppress logs.

**Review by robrap — COMMENTED** [review]
Reviewers: Ignore the quality issue.  It is a temporary isort problem because edx-drf-extensions is temporarily pointing to github.

**Comment by douglashall** [comment]
Is it not possible to modify the ecommerce alerting to ignore these info logs?

**Review by douglashall — APPROVED** [review]

**Comment by robrap** [comment]
Closing in favor of https://github.com/edx/ecommerce/pull/1983 which switches the order of the jwt_decode_handlers.

**Pull request closed without merging**